packages/jss/src/RuleList.js

```diff
@@ -127,16 +138,36 @@ export default class RuleList {
   /**
    * Update the function values with a new data.
    */
-  update: Update = (name, data) => {
+  update(...args: UpdateArguments): void {
```

Comment:
previous implementation was generating arrow method, which transpiles to a less performant inlined function in constructor

Reply:
👍